packages/mock-server/src/schema/Transaction.js

```diff
@@ -1,22 +1,43 @@
 import faker from 'faker';
 
+const choose = options => {
+  const numberOfOptions = options.length;
+
+  const randomIdx = Math.floor(Math.random() * numberOfOptions);
+
+  return options[randomIdx];
+};
+
 const TransactionData = Array.from({ length: 500 }).map((_, i) => ({
   id: `${i}`,
-  customer: `${faker.name.firstName()} ${faker.name.lastName()}`,
-  supplier: `${faker.name.firstName()} ${faker.name.lastName()}`,
-  date: faker.date.past().toString(),
-  total: `${faker.commerce.price()}`,
+  name: `${faker.name.firstName()} ${faker.name.lastName()}`,
```

Comment:
this is great! the new fields make the list look really good, and the randomness of the status is a nice touch.